5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
> @hanzz the OpenShift SCC defaults to run container as not root for very good reasons. I think the same reasons applies to the Atomic, @rhatdan ?

And I agree with you here, the application in the container should run as non-root. The container installed by the "atomic install" command still defaults  to non-root user. It is only the installation process running as root for the reasons I've mentioned earlier. It is question for the Project Atomic developers if they see a way how to do that as non-root or if it's useful to do it that way.